client/state/notices/middleware.js

```diff
@@ -141,6 +146,36 @@ export const onPublicizeConnectionUpdateFailure = ( dispatch, { error } ) => dis
 	} ) )
 );
 
+export const onThemeUploadSuccess = ( dispatch, { themeId } ) => {
+	return dispatch( successNotice(
+		translate( 'Successfully uploaded theme %(name)s', {
+			args: {
+				name: themeId
```

Comment:
Slight regression here in that before, we used the theme name if we had it available.